modules/data.remote/R/SMAP_SMP_prep.R

```diff
@@ -72,7 +72,7 @@ SMAP_SMP_prep <- function(site_info, start_date, end_date, time_points,
   time_points <- time_points[which(lubridate::year(time_points)>=2015)] #filter out any time points that are before 2015
   #initialize SMAP_Output
   SMAP_Output <- matrix(NA, length(site_info$site_id), 2*length(time_points)+1) %>% 
-    `colnames<-`(c("site_id", paste0(time_points, "_SoilMoistFrac"), paste0(time_points, "_SD"))) %>% as.data.frame()#we need: site_id, LAI, std, target time point.
+    `colnames<-`(c("site_id", paste0(time_points, "_SoilMoist"), paste0(time_points, "_SD"))) %>% as.data.frame()#we need: site_id, LAI, std, target time point.
```

Comment:
I'm surprised to see changes in the SMAP code as I thought you were working with a different product and because here you're changing the output name without changing any of the calculations. Does this imply that SMAP was already column sums not % and we had the units wrong? That would be surprising.

Reply:
The new variable "SoilMoist" was for the latest SMAP version 7 L4 data as there is only volumetric soil moisture but no soil wetness index in this dataset. The corresponding "SMAP.csv" was generated from Dongchen's latest codes here: https://github.com/DongchenZ/pecan/commit/193b2d05706cdd1f180fe0bfb4bc4643ee220ee8. So we might want to wait for his PR to be merge in later.